site/src/api/types.ts

```diff
@@ -14,3 +14,9 @@ export interface ReconnectingPTYRequest {
 export type WorkspaceBuildTransition = "start" | "stop" | "delete"
 
 export type Message = { message: string }
+
+// Keep up to date with coder/codersdk/features.go
```

Comment:
Is this something that can be autogenerated?

Reply:
My understanding is there's some reason why we can't get enums in our generated types.